lib/css/components/_stacks-toggle-switches.less

```diff
@@ -0,0 +1,89 @@
+//
+//  STACK OVERFLOW
+//  TOGGLE SWITCHES
+//
+//  This CSS comes from Stacks, our CSS & Pattern library for rapidly building
+//  Stack Overflow. For documentation of all these classes and how to contribute,
+//  visit https://stackoverflow.design/
+//
+//  ============================================================================
+//  $   BASE STYLE
+//  ----------------------------------------------------------------------------
+.s-toggle-switch {
+    display: flex;
+    align-items: stretch;
+
+    //  First hide the radio buttons
+    input[type="radio"] {
+        position: absolute;
+        left: -999em;
+        opacity: 0;
+    }
+
+    //  Next modifications to option cells
+    label {
+        //  Style the base label
+        width: 100%;
+        margin: 0;
+        padding: 0.8em 1em;
+        border: 1px solid var(--black-150);
+        background-color: @white;
+        color: var(--black-500);
+        font-size: @fs-body1;
+        font-weight: 400;
+        line-height: 1;
+        text-align: center;
+        white-space: nowrap;
+        cursor: pointer;
+          -webkit-user-select: none;
+             -moz-user-select: none;
```

Comment:
We can drop `-moz` and `-ms`. The only browser that needs a prefix is Safari

https://caniuse.com/#feat=user-select-none